tests/Avalonia.Controls.UnitTests/ComboBoxTests.cs

```diff
@@ -430,7 +430,8 @@ public void FlowDirection_Of_RectangleContent_Updated_After_OpenPopup()
                     {
                         new ComboBoxItem()
                         {
-                            Content = parentContent.Child
+                            Content = parentContent.Child,
+                            Template = null // ugly hack, so we can "attach" same child to the two different trees
```

Comment:
This hack replicates old test behavior, when there was no template on the ComboBoxItem. Now there is one always.